docs/tutorials/pilot_tutorial_demo.rst

```diff
@@ -0,0 +1,103 @@
+How to create a new Qunicorn-Pilot
+==================================
+
+1. Create a new python file in the `qunicorn_core.core.pilotmanager` directory. The qunicorn_pilot.py
+
+2. For the creation you can also just copy the base_pilot.py file and rename it. Keep in mind that the supported_languages and provider_name is set in the implement your pilot chapter.
+    .. image:: ../resources/images/tutorial_demo/create_pilot.png
+
+3. After at least creating the Pilot class inside extending from the base_pilot add your pilot to the "PILOTS" list in the pilotmanager.py file.
```

Comment:
"After creating a Pilot class extending from the base_pilot in the file, add your..." Possibly a nicer formulation?

Reply:
Better, but logically not quite correct - changed the old formulation